docs/tutorials/Tutorial_1d2d_fluvial_flood_model_for_Magdalena_with_answers.ipynb

```diff
@@ -1,1063 +1,1232 @@
lLEULM+dx5551z9uvr6+O2224jkUjQ1tbGBz7wAXzfn1PmwQcf5KqrrsJxHDZs2MDdd9993Pk/85nPsGbNGmKxGNu2bePhhx+e83utVuOd73wnzc3NpFIpXv/61x+XS+JkbTlw4MBx7Zj5fO1rX5vd70S/f+UrXznnbfnoRz96XD22bNmyoGuxVO7LqdoyPj7Ou971LjZv3kw8Hqenp4d3v/vdTE1NzTnHUrkvEScm0o6l84xG2rHMteP8xh07Nzz88MN6zZo1+rLLLtO/93u/N7v9pptu0m9729v0wMDA7OfYqGy+7+tLLrlE33zzzfqJJ57Q3/nOd3RLS4u+6667Zsvs27dPJxIJ/b73vU8/++yz+u/+7u+0YRj6e9/73myZr3zlK9q2bf2P//iP+plnntFve9vbdC6X00NDQ7Nl7rzzTr169Wp933336UcffVRfe+21+rrrrptXW3zfn9OGgYEB/bGPfUynUildLBZn9wX05z//+TnlqtXqOW/LRz7yEX3xxRfPqcfIyMi8919K9+VUbXnqqaf06173Ov3Nb35T79mzR993331648aN+vWvf/2ce7pU7kvE8UTaEbJUntFIO5a3dqz4DkexWNQbN27U9957r77pppuOE41jv7+Q73znO1pKqQcHB2e3ffazn9WZTEbX63WttdYf/OAH9cUXXzxnv9tvv13fcssts9+vueYa/c53vnP2exAEuqurS3/iE5/QWms9OTmpLcvSX/va12bL7Ny5UwP6oYcemldbXsgVV1yh77jjjjnbAP2Nb3zjpPucq7Z85CMf0ZdffvkJ6zCf/ZfSfTlVW07EV7/6VW3btvY8b3bbUrkvEXOJtOMoS+UZjbRjeWvHil9Seec738ltt93GzTfffMLfv/SlL9HS0sIll1zCXXfdRaVSmf3toYce4tJLL6W9vX122y233EKhUOCZZ56ZLfPCY99yyy089NBDALiuy2OPPTanjJSSm2++ebbMY489hud5c8ps2bKFnp6e2TLzacsMjz32GDt27OB3fud3Tng9WlpauOaaa/jHf/zHOWmIz2Vbdu/eTVdXF+vWrePNb34zfX19895/qd2Xk7XlRMyEGTbNuXkTl8p9iThKpB3HX4+l8IxG2rF8taPh6enPJ1/5yld4/PHHeeSRR074+5ve9CZ6e3vp6uriySef5EMf+hC7du3i61//OgCDg4NzHkxg9vvg4OApyxQKBarVKhMTEwRBcMIyzz333OwxbNsml8sdV2bmPKdry7F87nOfY+vWrVx33XVztn/84x/n5S9/OYlEgu9///u84x3voFQq8e53v/uctmXbtm3cfffdbN68mYGBAT72sY9xww038PTTT89r/6V0X07VlnQ6PWe/0dFR/viP/5i3v/3tc7YvlfsScZRIOyLtOJ9tWanasWI7HIcOHeL3fu/3uPfee4nFYicsc+zNu/TSS+ns7OQVr3gFe/fuZf369eeqqqdlPm2ZoVqt8uUvf5kPf/jDx/127LYrr7yScrnMX/zFX8w+nOeKW2+9dfbfl112Gdu2baO3t5evfvWrxOPxc1qXs+VUbTl2lFgoFLjtttu46KKL+OhHPzrnGEvlvkSERNoRace54ELUjhW7pPLYY48xPDzMVVddhWmamKbJD3/4Q/72b/8W0zQJguC4fbZt2wbAnj17AOjo6DjOEnfme0dHxynLZDIZ4vE4LS0tGIZxwjLHHsN1XSYnJ09YZiFt+Zd/+RcqlQq/+Zu/edprtG3bNg4fPky9Xj9nbTkRuVyOTZs2sWfPnnntv1Tuy+naMkOxWOSXfumXSKfTfOMb38CyrBPuO8NSuS8XKpF2RNpxvtsyw0rTjhXb4XjFK17BU089xY4dO2Y/L3rRi3jzm9/Mjh07MAzjuH127NgBQGdnJwDbt2/nqaeeYnh4eLbMvffeSyaT4aKLLpotc9999805zr333sv27dsBsG2bq6++ek4ZpRT33XffbJmrr74ay7LmlNm1axd9fX1s3759QW353Oc+x6/8yq/Q2tp62mu0Y8cO8vn8bHKfc9GWE1Eqldi7dy+dnZ3z2n+p3JfTtQXC0cmrXvUqbNvmm9/85mlHmbB07suFSqQdkXac77bACtWOBZmYLnOOtc7es2eP/vjHP64fffRRvX//fv1v//Zvet26dfrGG2+cLT/jQvWqV71K79ixQ3/ve9/Tra2tJ3Sh+sAHPqB37typP/OZz5zQ7chxHH333XfrZ599Vr/97W/XuVxujqX0nXfeqXt6evT999+vH330Ub19+3a9ffv2ebVlht27d2shhP7ud797XPlvfvOb+n/8j/+hn3rqKb17927993//9zqRSOg/+qM/Oudtef/7368ffPBBvX//fv2Tn/xE33zzzbqlpUUPDw/Pa/+ldF9O1ZapqSm9bds2femll+o9e/bMcV3zfX/J3ZeIkxNpx9J4RiPtWN7accF2OPr6+vSNN96om5qatOM4esOGDfoDH/jAHF96rbU+cOCAvvXWW3U8HtctLS36/e9//xy3JK21fuCBB/QVV1yhbdvW69at05///OePO/ff/d3f6Z6eHm3btr7mmmv0z372szm/V6tV/Y53vEPn83mdSCT0a1/7Wj0wMDCvtsxw11136dWrV+sgCI4r/93vfldfccUVOpVK6WQyqS+//HL9D//wD8eVPRdtuf3223VnZ6e2bVt3d3fr22+/Xe/Zs2dB12Kp3JdTteWBBx7QwAk/+/fvX3L3JeLkRNqxNJ7RSDuWt3ZE6ekjIiIiIiIiGs6KteGIiIiIiIiIWDpEHY6IiIiIiIiIhhN1OCIiIiIiIiIaTtThiIiIiIiIiGg4UYcjIiIiIiIiouFEHY6IiIiIiIiIhhN1OCIiIiIiIiIaTtThiIiIiIiIiGg4UYcjIiIiIiIiouFEHY6IiIiIiIiIhhN1OCIiIiIiIiIaTtThiIiIiIiIiGg4/z+hTyfOU7fdYQAAAABJRU5ErkJggg==",
```

Comment:
Woah, what happened here? Did you maybe run the notebook and then committed all changes?

Reply:
It's the output image.